src/esp/bindings_js/utils/vr_utils.js

```diff
@@ -0,0 +1,71 @@
+const VIEW_SENSORS = ["left_eye", "right_eye"];
```

Comment:
There's a lot of duplication here with the existing `esp/bindings_js/modules/vr_demo.js`. We should discuss offline.

Reply:
For the record: `vr_demo.js` will be deleted in https://github.com/facebookresearch/habitat-sim/pull/1366. The code duplication here is temporary.